C++/Canteen Management System/README.md

```diff
@@ -0,0 +1,38 @@
+# Canteen-Management
+
+This is a Simple Command Line User Interface program that is implemented using C++ . 
+This program is implemented using the concept of Binary Files.
+
+## Walk through video
+<a href="https://youtu.be/bWqX-9pGY-w">Click here</a>
```

Comment:
don't upload youtube link instead of that follow this steps to upload the video :

How to upload video in markdown then watch this video : 
[https://drive.google.com/file/d/12cjBwDaTCK4pu7cqNI_FNuRAZwamCywP/view?usp=sharing](https://drive.google.com/file/d/12cjBwDaTCK4pu7cqNI_FNuRAZwamCywP/view?usp=sharing&authuser=0)

**Note** : Screen Recorded video should include 
- code go-through 
- what are the files you have included
- run your project in screen recording